package/crds/iam.aws.upbound.io_roles.yaml

```diff
@@ -77,6 +77,27 @@ spec:
                     description: Whether to force detaching any policies the role
                       has before destroying it. Defaults to false.
                     type: boolean
+                  inlinePolicy:
+                    description: Configuration block defining an exclusive set of
+                      IAM inline policies associated with the IAM role. See below.
+                      Configuring one empty block (i.e.
```

Comment:
The description seems to be cut off

Reply:
We have some filtering mechanisms in the scraper. It filters some keywords. I think the cut-off reason is this. One option, by using configuration, is to override the description.

```go
r.MetaResource.ArgumentDocs["inline_policy"] = `Configuration block defining an exclusive set of IAM inline policies associated with the IAM role. See below. If no blocks are configured, Crossplane will not manage any inline policies in this resource. Configuring one empty block (i.e., inline_policy {}) will cause Crossplane to remove all inline policies added out of band on apply.`
```

cc @turkenf

Reply:
Thanks for the suggestion, I have added the configuration.